build.sh

```diff
@@ -2,7 +2,7 @@
 ## START STANDARD SITE BUILD SCRIPT INCLUDE
 readonly THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 readonly BOOTSTRAP="$(dirname "$THIS_SCRIPT")/resources/bootstrap.inc.sh"
-readonly BOOTSTRAP_VERSION=v0.11
+readonly BOOTSTRAP_VERSION=fix/keyman-hosts-for-server  # v0.11
```

Comment:
When keymanapp/shared-sites#53 lands
```suggestion
readonly BOOTSTRAP_VERSION=v0.17
```